pkg/driver/driver.go

```diff
@@ -84,6 +84,10 @@ func NewDriver(c cloud.Cloud, o *Options, m mounter.Mounter, md metadata.Metadat
 	case AllMode:
 		driver.controller = NewControllerService(c, o)
 		driver.node = NewNodeService(o, md, m, k)
+	/*TODO is the default case even possible to occur if we have a node mode that is not controller driver or all it will return
+	an error on line 71 as ValidateDriverOptions calls to ValidateMode which only allows the contoller node and all modes or it will
+	return an error
+	*/
```

Comment:
Start of discussion thread

Reply:
It shouldn't be but we should keep the default case as a defense in depth measure in case of bugs in the parsing code.